### PR TITLE
Add playwright test for login

### PR DESCRIPTION
**Pull request by sweta1308**
### Describe your changes:

Add tests in Playwright for Login spec.

#
### Type of change:

- [x] Improvement

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Login.spec.ts [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+  test('Signin using invalid credentials', async ({ page }) => {
+    await page.goto(`/signin`);
+    // Login with invalid email
+    await page.fill('#email', invalidEmail);
+    await page.fill('#password', CREDENTIALS.password);
+    await page.locator('.ant-btn:has-text("Login")').click();
+
+    await expect(
+      page.locator('[data-testid="login-error-container"]')
+    ).toBeVisible();
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Login.spec.ts:93]: We can remove this

File: openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Login.spec.ts [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    ).toHaveText(LOGIN_ERROR_MESSAGE);
+
+    // Login with invalid password
+    await page.fill('#email', CREDENTIALS.email);
+    await page.fill('#password', invalidPassword);
+    await page.locator('.ant-btn:has-text("Login")').click();
+
+    await expect(
+      page.locator('[data-testid="login-error-container"]')
+    ).toBeVisible();
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Login.spec.ts:105]: Same here

File: openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Login.spec.ts [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    await expect(page.getByTestId('success-line')).toHaveText(
+      LOGIN_ERROR_MESSAGE
+    );
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Login.spec.ts:108]:
```suggestion
    await expect(
      page.locator('[data-testid="login-error-container"]')
    ).toHaveText(LOGIN_ERROR_MESSAGE);
```

File: openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Login.spec.ts [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+  });
+
+  test('Forgot password and login with new password', async ({ page }) => {
+    await page.goto('/');
+    // Click on Forgot button
+    await page.locator('[data-testid="forgot-password"]').hover();
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Login.spec.ts:114]: Let's remove this

File: openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Login.spec.ts [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+
+    await expect(page).toHaveURL(`/forgot-password`);
+
+    // Enter email
+    await page.locator('#email').fill(CREDENTIALS.email);
+    // Click on Forgot button
+    await page.locator('.ant-btn:has-text("Submit")').click();
+    await page
+      .locator('[data-testid="go-back-button"]')
+      .scrollIntoViewIfNeeded();
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Login.spec.ts:125]:
```suggestion
```

File: openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Login.spec.ts [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+
+  test('Signup and Login with signed up credentials', async ({ page }) => {
+    await page.goto('/');
+
+    await expect(page).toHaveURL(`/signin`);
+
+    // Click on create account button
+    await page.locator('[data-testid="signup"]').scrollIntoViewIfNeeded();
+
+    await expect(page.locator('[data-testid="signup"]')).toBeVisible();
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Login.spec.ts:42]:
```suggestion
    
```

File: openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Login.spec.ts [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+      `${CREDENTIALS.firstName}${CREDENTIALS.lastName}`
+    );
+  });
+
+  test('Signin using invalid credentials', async ({ page }) => {
+    await page.goto(`/signin`);
+    // Login with invalid email
+    await page.fill('#email', invalidEmail);
+    await page.fill('#password', CREDENTIALS.password);
+    await page.locator('.ant-btn:has-text("Login")').click();
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Login.spec.ts:89]: we can utilise the data test id or id here.

File: openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Login.spec.ts [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+    // Click on Forgot button
+    await page.locator('[data-testid="forgot-password"]').hover();
+    await page.locator('[data-testid="forgot-password"]').click();
+
+    await expect(page).toHaveURL(`/forgot-password`);
+
+    // Enter email
+    await page.locator('#email').fill(CREDENTIALS.email);
+    // Click on Forgot button
+    await page.locator('.ant-btn:has-text("Submit")').click();
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Login.spec.ts:122]: please use id/data-testid

File: openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Login.spec.ts (hunk 1 above)
Comment (sweta1308) [reply openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Login.spec.ts]: Updated!

**Review by Sachin-chaurasiya — APPROVED** [review]

**Review by ShaileshParmar11 — APPROVED** [review]

**Pull request merged**